devices/Pcd8544/samples/Pcd8544.samples/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("CSharp.BlankApplication")]
```

Comment:
Use minimal version of that file the same as for library, we don't need the rest of attributes